packages/ai-bot/lib/ai-billing.ts

```diff
@@ -58,6 +59,25 @@ export async function saveUsageCost(
   }
 }
 
+export async function getAvailableCredits(
+  pgAdapter: PgAdapter,
+  matrixUserId: string,
+) {
+  let user = await getUserByMatrixUserId(pgAdapter, matrixUserId);
+
+  if (!user) {
+    throw new Error(
+      `should not happen: user with matrix id ${matrixUserId} not found in the users table`,
+    );
+  }
+
+  let availableCredits = await sumUpCreditsLedger(pgAdapter, {
+    userId: user.id,
+  });
```

Comment:
In the interest of efficiency, should we consider collapsing the above queries into a single query with a join?